README.md

```diff
@@ -1,3 +1,270 @@
+[checkCrossOrigin](https://github.com/fengyuanchen/cropperjs#checkcrossorigin) option.
+
+More info on the crossOrigin attribute:
+https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attr-crossorigin
+
+### data
+* Type: `object`
+* Default: `null`
+
+https://github.com/fengyuanchen/cropperjs#setdatadata
+
+### dragMode
+* Type: `string`
+* Default: `crop`
+
+https://github.com/fengyuanchen/cropperjs#dragmode
+
+### enable
+* Type: `bool`
+* Default: `true`
+
+Setting the value to false will disable the component.
+
+https://github.com/fengyuanchen/cropperjs#enable
+https://github.com/fengyuanchen/cropperjs#disable
+
+### imgType
+* Type: `string`
+* Default: `image/jpeg`
+
+The image type to return when getting the cropped image data URL from the canvas using `toDataURL`.
+Valid options are image/png, image/jpeg, and image/webp. If the data URL returned shows that the format is PNG,
+then that means the image type submitted is not supported.
+
+https://developer.mozilla.org/en-US/docs/Web/API/HTMLCanvasElement/toDataURL
+
+### imgQuality
+* Type: `number`
+* Default: `0.92`
+
+The quality of the image to return when getting the cropped image data URL from the canvas using `toDataURL`.
+Should be a value between 0 and 1 (a percentage). If the value is anything else, the default value of 0.92 will be used.
+
+https://developer.mozilla.org/en-US/docs/Web/API/HTMLCanvasElement/toDataURL
+
+### moveTo
+* Type: `array`
+* Default: N/A
+
+```javascript
+[
+  100, // X value
+  200  // Y value
+]
+```
+
+https://github.com/fengyuanchen/cropperjs#moveto
+
+### ready
+* Type: `function`
+* Default: N/A
+
+https://github.com/fengyuanchen/cropperjs#ready-1
+
+### rotateTo
+* Type: `number`
+* Default: `0`
+
+https://github.com/fengyuanchen/cropperjs#rotateto
+
+### scaleX
+* Type: `number`
+* Default: `1`
+
+https://github.com/fengyuanchen/cropperjs#scalexscalex
+
+### scaleY
+* Type: `number`
+* Default: `1`
+
+https://github.com/fengyuanchen/cropperjs#scalexscaley
+
+### style
+* Type: `object`
+* Default: `null`
+
+Custom styles for the component.
+
+https://github.com/fengyuanchen/cropperjs#scalexscaley
+
+### zoom
+* Type: `function`
+* Default: N/A
+
+**Parameters**
+* event: Custom event object
+* dataURL: The cropped image data URL returned in the format specified in the imgType prop
+
+```javascript
+<ReactCropperJS
+  zoom={(event, dataURL)} => {
+    console.log(event.detail);
+    this.img.src = dataURL;
+  }}
+/>
+```
+
+https://github.com/fengyuanchen/cropperjs#zoom-1
+
+### zoomTo
+* Type: `number`
+* Default: `1`
+
+https://github.com/fengyuanchen/cropperjs#zoomto
+
+### Other props
+Accepts all options found in the CropperJS [docs](https://github.com/fengyuanchen/cropperjs#options) as props.
+Except for the props listed above, all props passed in are set only when the component is mounted and cannot
+be changed unless the component is unmounted and re-mounted.
+
+## License
+[MIT](https://opensource.org/licenses/MIT)
```

Comment:
My thought on reading this documentation was, "Wow, this is super annoying to have to open up all these links to figure out what these properties do."

Is there a better way to do this? Could we bring in the extra sentence from the documentation we link to so if that if it says like for the ZoomTo Example:

> Zoom the canvas (image wrapper) to an absolute ratio.

It'd save me a lot of clicks.

Or would it just be better to say, "Hey, we support this list of properties but to see how they work, go to this link and read through it all."

It seems like we should either have enough documentation to not require 50 links or just flat out admit we're not trying to document things here but here's what we let you do and here's where you can read in one link instead of 50 links.